x-pack/platform/plugins/shared/stack_connectors/public/connector_types/openai/connector.tsx

```diff
@@ -105,12 +116,100 @@ const ConnectorFields: React.FC<ActionConnectorFieldsProps> = ({ readOnly, isEdi
         />
       )}
       {config != null && config.apiProvider === OpenAiProviderType.Other && (
-        <SimpleConnectorForm
-          isEdit={isEdit}
-          readOnly={readOnly}
-          configFormSchema={otherOpenAiConfig}
-          secretsFormSchema={otherOpenAiSecrets}
-        />
+        <>
+          <SimpleConnectorForm
+            isEdit={isEdit}
+            readOnly={readOnly}
+            configFormSchema={otherOpenAiConfig}
+            secretsFormSchema={otherOpenAiSecrets}
+            configValidations={
```

Comment:
This prop does not exist on `SimpleConnectorForm`. What are you trying to do here?

Reply:
Fixed